build/tests/multi-user.sh

```diff
@@ -86,12 +86,13 @@ declare -i u p v value
 
 say increment the value with a simple expression ${iterations} times, querying enclaves in round robin
 for v in $(seq 1 ${iterations}) ; do
+    echo pass $v
     u=$((v % user_count + base_user))
     p=$((v % port_count + base_port))
     value=$(${PDO_HOME}/bin/pdo-invoke.psh \
                        --wait yes \
                        --logfile __screen__ --loglevel ${PDO_LOG_LEVEL} \
-                       --enclave "http://localhost:${p}" --identity user${u} \
+                       --enclave "http://${PDO_HOSTNAME}:${p}" --identity user${u} \
                        --pdo_file ${SAVE_FILE} --method anonymous_inc_value)
```

Comment:
So, I see that this allows client to sit on a different machine, but you assume that all eservices are located within PDO_HOSTNAME? nothing wrong with that, I guess that is an assumption for this test.

Reply:
pdo_hostname IS the name of the local server. it will be different and unique on each host. so it does not introduce any restrictions.